source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -45,6 +59,10 @@ bool BioXASSideAppController::setupDataFolder()
 void BioXASSideAppController::initializeBeamline()
 {
 	BioXASSideBeamline::bioXAS();
+	connect( BioXASSideBeamline::bioXAS()->ge32ElementDetector(), SIGNAL(connected(bool)), this, SLOT(onGeDetectorConnectedChanged()) );
+	onGeDetectorConnectedChanged();
```

Comment:
All UI related operations should only happen in the virtual functions called by `setupUserInterface()`, and beamline specific stuff should go to `setupUserInterfaceImplementation()`.